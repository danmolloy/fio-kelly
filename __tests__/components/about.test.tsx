import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import AboutSection from "../../components/about";
describe("<About />", () => {
  beforeEach(() => {
    render(<AboutSection />);
  });
  it("about-section is in the document", () => {
    const aboutSection = screen.getByTestId("about-section");
    expect(aboutSection).toBeInTheDocument();
  });
  it("matches snapshot", () => {
    const aboutSection = screen.getByTestId("about-section");
    expect(aboutSection).toMatchSnapshot();
  });
});
