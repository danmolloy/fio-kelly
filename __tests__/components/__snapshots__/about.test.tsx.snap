// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> matches snapshot 1`] = `
<div
  class=" w-full h-full  flex flex-row "
  data-testid="about-section"
>
  <article
    class="text-black tracking-wider lg:w-2/3"
  >
    <p
      class="my-2 "
    >
      Irish flautist Fiona Kelly has been hailed by the New York Times as a player with “impressive technique and elegant musicianship”.
    </p>
    <p
      class="my-2"
    >
      Based in London, Fiona is in demand as an orchestral and chamber musician. She leads a busy freelance career performing as guest principal flute with many of the UK’s leading orchestras including the London Philharmonic Orchestra, the Philharmonia, Royal Philharmonic Orchestra, the Academy of St Martin in the Fields and the Orchestra of the Royal Opera House, Covent Garden amongst others.
    </p>
    <p
      class="my-2"
    >
      Solo and chamber music engagements have led to collaborations with ensembles and artists including the Wigmore Soloists, Antje Weithaas, Uri Caine and Håkan Hardenberger. Fiona has performed concertos at the BBC Proms, Rheingau and Heidelberg festivals. With a keen interest in new music, Fiona has premiered and recorded new works by Uri Caine, Stephan Mackey and Ian Wilson.
    </p>
    <p
      class="my-2"
    >
      Fiona has a duo with her harpist sister Jean Kelly. They have performed recitals and concertos throughout the U.K. and Ireland. A recent highlight was a recital at the National Concert Hall Chamber Music Series: The Natural World, with a programme titled “Toward the Sea”. The Kelly Sisters often perform works and arrangements of Irish music by their grandfather, composer T.C Kelly.
    </p>
    <p
      class="my-2"
    >
      Fiona enjoys regularly returning home to Ireland to the Irish Chamber Orchestra, where she is guest principal flute. Before relocating to London, Fiona was principal flute of the Swedish Chamber Orchestra from 2012-2016.
    </p>
    <p
      class="my-2"
    >
      Beginning her studies in Cork with her mother Evelyn Grant and Sabine Ducrot, Fiona went on to complete her studies at Trinity Laban Conservatoire with Anna Noakes, and at The Juilliard School with Robert Langevin.
    </p>
    <p
      class="my-2"
    >
      Fiona is a flute professor at Trinity Laban Conservatoire and gives masterclasses at music departments and colleges throughout the UK and Ireland.
    </p>
  </article>
  <div
    class="hidden lg:inline mx-8"
  >
    <img
      alt="Grayscale silhouette of Fiona holding her flute"
      data-nimg="1"
      decoding="async"
      height="429"
      loading="lazy"
      src="/_next/image?url=%2Fimages%2Ffk-bw-flute.jpg&w=750&q=75"
      srcset="/_next/image?url=%2Fimages%2Ffk-bw-flute.jpg&w=384&q=75 1x, /_next/image?url=%2Fimages%2Ffk-bw-flute.jpg&w=750&q=75 2x"
      style="color: transparent;"
      width="363"
    />
  </div>
</div>
`;
